mkdocs.yml

```diff
@@ -21,6 +21,7 @@ nav:
       - Server: pages/Server.md
       - Builder: pages/Builder.md
       - Project: pages/Project.md
+    - ECMAScript Version Support: pages/ESSupport.md
```

Comment:
- Miscellaneous (as of now Troubleshooting)
-- Troubleshooting
-- Benchmarking
-- ECMAScript Version Support

Reply:
done